sdk/python/foundation-models/system/inference/image-embeddings/image-embeddings-batch-endpoint.ipynb

```diff
@@ -0,0 +1,494 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image Embeddings Inference using Batch Endpoints\n",
+    "\n",
+    "This sample notebook shows how to deploy `embeddings` type models to a batch endpoint for image embeddings inference.\n",
+    "\n",
+    "### Task\n",
+    "The input to the `embeddings` task is images, text or image-text pairs, and the output is feature embeddings produced by the model. In this notebook, we are only concerned with image inputs.\n",
+    " \n",
+    "### Model\n",
+    "Models that can perform the `embeddings` task are tagged with `embeddings`. We will use the `Facebook-DinoV2-Image-Embeddings-ViT-Base` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../import/import_model_into_registry.ipynb) and then use them for inference. \n",
```

Comment:
nit: may be you can mention about model in catalog.

Reply:
Done.